flow/record/tools/rdump.py

```diff
@@ -3,9 +3,11 @@
 
 import logging
 import sys
+import urllib.parse as urlparse
 from importlib import import_module
 from pathlib import Path
 from textwrap import indent
+from urllib.parse import urlencode
```

Comment:
Can you change the urllib imports to:

```python
from urllib.parse import urlencode, urlparse
```

And then change the `urlparse.urlparse` code to just `urlparse`.

Reply:
Adjusted